src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,73 @@
+package duke;
+import java.util.Scanner;
+import java.time.LocalDate;
+
+public class Ui {
+    String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+    String toPrint = "Hello from\n" + logo + "\n Hello! I'm Duke\n What can I do for you?";
+
+    Ui() {}
+
+    /**
+     * Handling User Input
+     */
+    void getUserCommand(TaskList myTasks) {
+        Scanner scanner = new Scanner(System.in);
+        while (scanner.hasNextLine()) {
+            String userInput = scanner.nextLine();
```

Comment:
setting the input to uppercase letters and parsing the uppercased commands like "BYE" could help handle cases where users input commands like "Bye", "bYe", etc..